### DISCUSSION TITLE
Fails to find package.json files in a monorepo subdirectory

### DISCUSSION THREAD

**Discussion by runspired** [renovatebot/renovate#19339] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

https://github.com/emberjs/data is an OSS monorepo that uses renovate; however, it fails to detect the package.json files of the projects within the `tests/` subdirectory (it does find those within the `packages/` subdirectory).

Attempting to force discovery by supplying `includePaths` was unsuccessful, probably due to a lack of documentation on this feature leading to a shaky attempts at best to guess whether globs or roots was enough or if exact file paths were found (only exactly specified filers were found).

Scanning through the logs, it appears that renovate applies some default `ignorePaths` which we need to override; however, it is not clear from the documentation if this is even possible.

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Found repo ignorePaths
{
  "ignorePaths": [
    "**/node_modules/**",
    "**/bower_components/**",
    "**/vendor/**",
    "**/examples/**",
    "**/__tests__/**",
    "**/test/**",
    "**/tests/**",
    "**/__fixtures__/**"
  ]
}
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by runspired** [comment, marked as answer]
I notice there's a footnote in the docs that this is a result of extending `config:base`, if it turns out `ignorePaths` can't be overridden then eliminating that should work.

**Reply by runspired** [reply]
Looks like replacing that was enough, sorry for the issue, but also thank you for the helpful issue triage flow: it taught me about the debug logs which I never would have found otherwise.

**Comment by viceice** [comment]
tests are ignored by default, you need to exclude the following preset and set your own ignorePaths.

https://docs.renovatebot.com/presets-default/#ignoremodulesandtests
